robot/EDA/tests/browser/settings/eda_settings_accounts_and_contacts_edit_mode.robot

```diff
@@ -14,15 +16,10 @@ Test Checkbox
 
 Validate Edit Mode For Accounts and Contacts Settings
     [tags]                          unstable
-    Go To Eda Settings
+    Go to page                      Custom      HEDA_Settings
```

Comment:
I like this change!

Reply:
Help from Sravani again! I see that we need to do this implementation throughout.